README.md

```diff
@@ -27,10 +27,15 @@ Or install it yourself as:
 Add a new initializer `config/initializers/librato.rb`:
```

Comment:
I think this should also change from 'Add a new initializer' to 'Update the `config/initializers/metrics.rb`'. Otherwise because 'M' comes after 'L' the metrics initializer will overwrite the librato initializer.